Orange/widgets/data/owimpute.py

```diff
@@ -563,8 +563,13 @@ def column_imputer_minimal(variable, table):
 
 
 def column_imputer_average(variable, table):
-    stats = basic_stats.BasicStats(table, variable)
-    return column_imputer_defaults(variable, table, stats.mean)
+    if variable.is_continuous:
```

Comment:
The function is called `column_imputer_average` so I think it should do that and not switch to something else for discrete variables.

The function below was obviously intended for the functionality of the else clause, but is currently broken (no return) and not used.
I think this should be fixed instead.
E.g. have the if in `imputer_for_column` and call either `column_imputer_average` (for cont) or `column_imputer_modus` (for disc).